test-support/src/main/java/org/springframework/ldap/test/EmbeddedLdapServer.java

```diff
@@ -85,7 +104,50 @@ public static EmbeddedLdapServer newEmbeddedServer(String defaultPartitionName,
     public void shutdown() throws Exception {
         ldapServer.stop();
         directoryService.shutdown();
-
-        FileUtils.deleteDirectory(workingDirectory);
     }
+
+	private static void initSchemaPartition(DefaultDirectoryService directoryService)
```

Comment:
This is required, without it ldap can not initialize. This method initializes the schema manager and add the schema partition to directory service.